hera_qm/vis_metrics.py

```diff
@@ -54,13 +54,10 @@ def check_noise_variance(data):
     return Cij
 
 
-def sequential_diff(data, t_int=None, axis=(0,), pad=True, history=''):
+def sequential_diff(data, t_int=None, axis=(0,), pad=True, run_check=True, history=''):
     """
     Take a sequential (forward) difference of a visibility waterfall
```

Comment:
Can you make this "one-line summary" actually fit on a single line? When the short-form of the docstring is requested, only the first line is returned, so it's helpful for the summary to fit on a single line.